scrunch/datasets.py

```diff
@@ -1393,6 +1396,16 @@ def __repr__(self):
     def __str__(self):
         return self.name
 
+    _categories = None
+
+    @property
+    def categories(self):
+        if self.resource.type not in self.CATEGORICAL_TYPES:
```

Comment:
This is one of the cases where I'm not sure if we should internally use self.type or if it is ok to use self.resource.type.